src/components/Review.jsx

```diff
@@ -1,17 +1,45 @@
-import { Comments } from "./Comments";
+import axios from "axios";
+import { useState } from "react";
 
 export function Review({ title, designer, owner, review_img_url, category, created_at, votes, review_body, review_id }) {
+  const [currentVotes, setCurrentVotes] = useState(votes);
+  const [isDisabled, setIsDisabled] = useState(false);
+
+  function handleVoting(e) {
+    const incVotes = e.target.classList.contains("UpVote") ? 1 : -1;
+    axios
+      .patch(`https://be-nc-games.onrender.com/api/reviews/${review_id}`, { inc_votes: incVotes })
+      .then((response) => {
```

Comment:
Introduce an api.js to this project repo to extract away the axios requests. 

Optional: This will also mean you can create a base API URL with axios in that file (using axios.create)